Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -336,6 +336,8 @@
       - id: HoSIDCard # Delta-V
       - id: LunchboxCommandFilledRandom # Delta-V Lunchboxes!
         prob: 0.3
+#LPP
```

yamllint

[warning] missing starting space in comment

(comments)